InternTrack.Core.Api.Tests.Acceptance/Brokers/ApiTestCollection.cs

```diff
@@ -0,0 +1,13 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+using Xunit;
+
```

Comment:
Same here extra line and add the line above.

Reply:
I will look into it . Thank you